iModelCore/iModelPlatform/DgnCore/DgnModel.cpp

```diff
@@ -624,6 +624,48 @@ DgnDbStatus DefinitionModel::_OnInsertElement(DgnElementR el)
     return el.IsInformationContentElement() ? T_Super::_OnInsertElement(el) : DgnDbStatus::WrongModel;
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+SheetIndexModelPtr SheetIndexModel::Create(DgnDbR db, DgnElementId modeledElementId)
+    {
+    ModelHandlerR handler = dgn_ModelHandler::SheetIndex::GetHandler();
+    DgnClassId classId = db.Domains().GetClassId(handler);
+    DgnModelPtr model = handler.Create(DgnModel::CreateParams(db, classId, modeledElementId));
+    if (!classId.IsValid() || !model.IsValid())
+        {
+        BeAssert(false);
+        return nullptr;
+        }
+
+    return dynamic_cast<SheetIndexModelP>(model.get());
+    }
+
+// /*---------------------------------------------------------------------------------**//**
+// * @bsimethod
+// +---------------+---------------+---------------+---------------+---------------+------*/
+SheetIndexModelPtr SheetIndexModel::Create(SheetIndexPartitionCR modeledElement)
+    {
+    return SheetIndexModel::Create(modeledElement.GetDgnDb(), modeledElement.GetElementId());
+    }
+
+// /*---------------------------------------------------------------------------------**//**
+// * @bsimethod
+// +---------------+---------------+---------------+---------------+---------------+------*/
+SheetIndexModelPtr SheetIndexModel::CreateAndInsert(SheetIndexPartitionCR modeledElement)
+    {
+    SheetIndexModelPtr model = Create(modeledElement);
+    return (model.IsValid() && (DgnDbStatus::Success == model->Insert())) ? model : nullptr;
+    }
+
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+DgnDbStatus SheetIndexModel::_OnInsertElement(DgnElementR element)
+    {
+    return element.IsInformationContentElement() ? T_Super::_OnInsertElement(element) : DgnDbStatus::WrongModel;

```

Comment:
This implementation is identical to that of the [superclass method](https://github.com/iTwin/imodel-native/blob/900ca418ffc8e2c1c34156bfa2356f9f1492d49e/iModelCore/iModelPlatform/DgnCore/DgnModel.cpp#L562).

Reply:
Function removed.